proposals/0047-last-hard-fork-syscall.md

```diff
@@ -0,0 +1,120 @@
+---
+simd: '0047'
+title: Syscall to get the last hardfork
+authors:
+  - Godmode Galactus (Mango Markets)
+category: Standard
+type: Core
+status: Draft
+created: 2023-04-15
+---
+
+## Summary
+
+Create a new sysvar `SysvarLastHardFork1111111111111111111111111`, which can
+be used to get the last hard fork slot.
```

Comment:
The sysvar can be skipped entirely, this only needs to be a syscall who's signature can be described here instead.